trace.go

```diff
@@ -15,12 +15,14 @@
 package stackdriver
 
 import (
+	"context"
 	"fmt"
 	"log"
 	"sync"
 	"time"
 
 	tracingclient "cloud.google.com/go/trace/apiv2"
+	"github.com/gogo/protobuf/proto"
```

Comment:
What's the rationale behind using this instead of https://godoc.org/github.com/golang/protobuf/proto?